feat(blog): add blog post about gather & metaverse

georgiee opened the pull request:
This is a new blog post about Gather and the Metaverse. Please have a read. I checked the grammar with the free edition of grammarly. I also have created a teaser image by using the mapmaker plus a little bit of photoshop, that was a pain in the ass so please don't have any issues with that 😁

Preview Blog Post:
https://satellytescommain21751-gkbloggather.gtsb.io/blog/gather-metaverse-bump-each-other-2021/

Preview on metatags.io doesn't work. I guess the 500 error again. We need to check this before migrating to Gatsby beyond the branch previews.

## How to review
Please read, thank you 🙏

---
Some collected issues to handle through #251 maybe?

+ Adding a video from local files requires a manual video html snippet
+ Gif files are not processed (which might be okay) but the size is not over the full width
+ I would like to embed youtube but I can't do that easily. In addition I would like to make it privacy first and cache an image locally so people don't communicate with youtube & co. directly.
+ I would like to use an expandable or such but I can't embed custom components for this purpose
+ I would like to use our callout but I can't yet.
+ I would like to create images with real captions. Right now there is just a `em` below.

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**satellytes.com-main**

:tada: Your build was successful! [See the Deploy preview here.](https://satellytescommain21751-gkbloggather.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/674be368-adf0-4ac5-8fae-f9a4493adc51/builds/fc71d29d-5819-4f3f-8b86-faa0c077b3d5/details)

:clock1: Build time: **1m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 70   |
| Accessibility |   :green_heart: 98   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 79   |

:link: [View full report](https://build-fc71d29d-5819-4f3f-8b86-faa0c077b3d5.gtsb.io/reports/lighthouse/index.html)

georgiee commented:
FYI I've pushed a temporary change to the sharing images in order to test the generation behavior from gatsby and if we can switch to images for blog posts.

georgiee commented:
Good:
https://satellytescommain21751-gkbloggather.gtsb.io/static/8397ca28c2a71074cdf29741cf17f985/3ccee/teaser.png

Bad:
https://satellytescommain-gkbloggather.gtsb.io/static/8397ca28c2a71074cdf29741cf17f985/3ccee/teaser.png

The base branch is not generated correctly. I will investigate tomorrow

georgiee committed:
feat: add blog post

georgiee commented:
I have reverted all technical changes. Just the blog post again👍

chunksnbits commented on blog-posts/gather-metaverse-bump-each-other-2021.md:
```diff
@@ -0,0 +1,130 @@
+## Introduction
+A few weeks ago, we introduced [Gather](https://www.gather.town/) at Satellytes to make our daily remote life more pleasant and to fight [Zoom Fatigue](https://en.wikipedia.org/wiki/Zoom_fatigue). Gather describes itself as a **_"platform that combines a 2D game-like interface with video-chat"_**. 
+
+![Official gather screenshot with people hanging around](images/gather/gather-example-official.png)
```
I think I would push this back a little bit. I personally found this hindering me getting into the article.

georgiee replied:
Thanks, I moved it down to the entire introduction. I still want to give a first image to the reader's mind.

![Screen Shot 2021-12-22 at 15 47 13](https://user-images.githubusercontent.com/1701755/147110523-df17ed03-cf41-4f92-9f11-4767c7d45e97.png)

chunksnbits commented on blog-posts/gather-metaverse-bump-each-other-2021.md:
```diff
@@ -0,0 +1,130 @@
++ I don't trust Gather that the microphone and video is truly switched off every time I focus on another window
++ I'm intimidated with Gather as I don't play games often
++ I just forget to start Gather on the morning
+
```
From my personal view: "It doesn't integrate in any way with my work routing / other tools" or "It requires me to spend too much focus on the app itself (the effect 'bumping' into someone is rather an active one)"

georgiee replied:
thanks I will add those issues

chunksnbits commented on blog-posts/gather-metaverse-bump-each-other-2021.md:
```diff
@@ -0,0 +1,130 @@
+## Conclusion
+Satellytes did not start as a remote-only company more than three years ago. People enjoyed going to the office and talking to each other. That's why our folks had an actual pain when we were forced to work remotely. On the other side, everyone feels the power of remote work, that we want to embrace in the future. 
+
+The Metaverse seems to be the glue we have been missing all the time. Gather is an excellent way to dive into the Metaverse. I got a big fan of Gather in the past weeks. I'm happily tinkering around with their [mapmaker](https://github.com/gathertown/mapmaking) and constantly improving our virtual space because I really want our folks to enjoy the Metaverse. Let's see where we are heading with th ...
```
```suggestion
The Metaverse seems to be the glue we have been missing all the time. Gather is an excellent way to dive into the Metaverse. I became a big fan of Gather in the past weeks. I'm happily tinkering around with their [mapmaker](https://github.com/gathertown/mapmaking) and constantly improving our virtual space because I really want our folks to enjoy the Metaverse. Let's see where we are heading with that experiment in 2022. 
```

georgiee replied:
thanks

georgiee replied:
I think the parsing from github is broken 😂 The highlighted words are escaped and not displayed with the adequate highlight.

![Screen Shot 2021-12-22 at 15 53 47](https://user-images.githubusercontent.com/1701755/147111435-3855e63a-3309-487e-bec9-cbfe6d554b65.png)

chunksnbits commented on blog-posts/gather-metaverse-bump-each-other-2021.md:
```diff
@@ -0,0 +1,130 @@
+
+_Official Gather video from [gather.town](http://www.gather.town)_
+
+### Daily
```
"Daily routine"? "Hanging around"? 
At first I though about the scrum artifact, when I read "Daily", tbh

georgiee replied:
good point 🙏

chunksnbits commented on blog-posts/gather-metaverse-bump-each-other-2021.md:
```diff
@@ -0,0 +1,130 @@
+![Official gather screenshot with people hanging around](images/gather/gather-example-official.png)
+_Official gather screenshot with people hanging around_
+
+It might look like a game, but it's only the optics. Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being ...
```
```suggestion
At first glance it might look like a game. But that's only the visual presentation chosen by gather (although you can run go-karts, if you like to 😄). The core use-cases are actually all focused around the office live. 

Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.
```
```suggestion
It might look like a game, but it's only the optics. Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.
```

georgiee replied:
what happened with your first suggestion? I see a lot of html mixed in ?

![Screen Shot 2021-12-22 at 15 51 46](https://user-images.githubusercontent.com/1701755/147111170-ccc613bc-895e-43e7-a8d8-6079ac3c2879.png)

chunksnbits commented on blog-posts/gather-metaverse-bump-each-other-2021.md:
```diff
@@ -0,0 +1,130 @@
+From the beginning, I forced myself to keep Gather running every day to act as a role model. Occasionally, I would invite someone from the team to try out Gather with me. 
+
+### All-hands meetings
+Once I reached a "critical mass" of people using Gather, I decided to switch our all-hands meeting on Mondays from Zoom to Gather. We would run our meeting on a small, sunny island. That was also the first time I had to use the [mapmaker](https://support.gather.town/help/mapmaker). Because everything is possible in the Metaverse I decided to build a suburban station to connect the island with our ...
```
```suggestion
Once we reached a "critical mass" of people using Gather, I decided to switch our all-hands meeting on Mondays from Zoom to Gather. We would run our meeting on a small, sunny island. That was also the first time I had to use the [mapmaker](https://support.gather.town/help/mapmaker). Because everything is possible in the Metaverse I decided to build a suburban station to connect the island with our office.
```

georgiee replied:
thanks

chunksnbits commented on blog-posts/gather-metaverse-bump-each-other-2021.md:
```diff
@@ -0,0 +1,130 @@
+![](images/gather/gather-smm.jpg)
+_Our Satellytes folks gathering for our all-hands meeting on a Monday_
+
+The actual meeting is comparable to Zoom or Meet. We broadcast some news and people can share thoughts and ask questions. The time before and after that meeting is where the Metaverse shines ✨. 
```
I personally think the existence of broadcasting spots is even one of the more interesting / useful details. 
I wondered in the beginning myself, how the spatial audio thing would work there

georgiee replied:
You are right. Actually we don't even need those because meeting rooms allow to speak to everyone else too. The spotlight is rather to reach everyone in every room. I misused that. I check if I can rewrite this part.

chunksnbits reviewed (commented):
Well done, I actually found this a good and interesting read (regardless of the intention to do some reviewing)! 
Generally I think the examples and use-cases are well picked. 
I also found it genuine enough to feel useful to others interested in the topic.

Personally I think though the use of "Metaverse" is a bit overdone. This is probably to increase the SEO impact, but maybe limit some instances (esp. in the last paragraph)?

fischeversenker reviewed (approved):
I enjoyed reading this :+1:

I "thumbsed up" some of the comments from @chunksnbits since I would have left similar comments. I especially like the idea to place the first image a little further down. Right now, looking at the first few lines it feels very cluttered:
![image](https://user-images.githubusercontent.com/7970458/147084326-75b7d73c-4ea1-4258-8ae1-6cc3c156c1a5.png)

Other than that I'm happily approving this. It's already ready for the public in my opinion 🙌

georgiee commented:
Thanks folks! I will update the content and I see very valid feedback. Very much appreciated 🙏

georgiee committed:
chore: review updates

georgiee commented:
Thanks again. I went over your feedback and accepted virtually every suggestion👍 I also added a few more words on the all-hands meeting description. I will auto merge this now as it's approved (and I really need to get this out of my mind). I'm still happy to fix things afterwards. I'm really thankful that you have read and given that nice feedback. Thank you!

I have also removed the triple mention of Metaverse in the conclusion. I really mentioned it quite a lot and SEO was not involved, it was me talking too much about something interesting 😅

georgiee committed:
chore: update
